src/pages/pu-vendor-groups/index.js

```diff
@@ -0,0 +1,122 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useWindow } from 'src/windows'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { PurchaseRepository } from 'src/repositories/PurchaseRepository'
+import VendorGroupsForm from './forms/VendorGroupsForm'
+
+const VendorGroups = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack } = useWindow()
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50 } = options
```

Comment:
this api has no paginations, please remove them and for next time check api if request pagination params or no